pkgs/by-name/st/strongswan/package.nix

```diff
@@ -5,6 +5,7 @@
 , systemd, pam
 , curl
 , enableTNC            ? false, trousers, sqlite, libxml2
+, enableTPM2           ? true, tpm2-tss
```

Comment:
Unsure if we should enable this by default;
For sake of hardening I would reduce secret providing plugins to user selected.